### PR TITLE
feat(action): auto-assign issues

### PR DESCRIPTION
**Pull request by moT01**
I'm not sure if this is something we want - I thought I would open it up and we can discuss. It will auto assign issues to two random users in the list - which should probably be expanded.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes ~~either locally on my machine, or GitPod~~ on github.

Closes #XXXXX

### CODE REVIEW

File: .github/workflows/assign-issue.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    types: [opened]
+
+jobs:
+  auto-assign:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Auto-assign issues'
+        uses: pozil/auto-assign-issue@v1.5.0
+        with:
+          assignees: ojeytonwilliams,ShaunSHamilton,nhcarrigan,moT01
```
[the remark this answers is not in the document]
Comment (Sembauke) [reply .github/workflows/assign-issue.yml:14]: Eh, be careful SHAUN! 😄

[the remark this answers is not in the document]
Comment (nhcarrigan) [reply .github/workflows/assign-issue.yml:14]: Can we not do it with the `dev-team`?

**Review by raisedadead — COMMENTED** [review]
No way!

**Comment by Sboonny** [comment]
From outsider perspective, I am against self assignment. I remember an old typo issue (solved) in relational databases, that I knew how to help, but stopped, because I saw Tom self assignment. I didn't know that it's acceptable to make pr until the first meeting I attended.

**Comment by moT01** [comment]
I should have mentioned this in the description - for some clarity, this is not for assignment of fixing issues. The purpose is that assigned users would guide the issues from needing triage to discussing and coming up with a solution, and either adding a help wanted label or closing it.

File: .github/workflows/assign-issue.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (moT01) [reply .github/workflows/assign-issue.yml:14]: yes, we could do a team @nhcarrigan - I wasn't sure who exactly we wanted - a number of people on the dev-team aren't all that active in this repo and/or maybe busy with other repos, so I didn't want to spam them with notifications. I plan to bring this up in the next meeting.

**Comment by raisedadead** [comment]
OK - all jokes and playfulness aside. Tom, I think what you are proposing is a helpful idea. The GitHub issue triaging is a big deal, and we often leave issue threads open-ended.

Instead of individual accountability, I propose that we use something like the "stale issues bot" (there is one on the GitHub marketplace) to "@" mention the moderators' team on issues that do not have a proper tag (like they are not triaged, or are in discussion for a long while).

That way, we can kindle interest or close issues that no one seems interested in.

**Comment by ahmadabdolsaheb** [comment]
Something Like this could help with notifications as well. Hear me out.

So every issue initially gets a moderator who could helps with setting up the tags, assigning another members(if the issue is not in the assignee's scope of expertise), or chose a tag (let's call it "processing" for now) to signal to others that the assignee does not know what to do what it or having a problem.

In that case dev team members can ignore most of the notification noise an only focus on issues that they are assigned to and issues that have that specific tag to help other members.

**Comment by Sembauke** [comment]
We decided to do a biweekly issue triage meeting instead.

**Pull request closed without merging**